client/js/screens/AuthWelcome/AuthWelcomeContainer.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import AuthWelcome from "./AuthWelcome";
+class AuthWelcomeContainer extends Component {
+  static navigationOptions = {
+    title: "SignIn"
```

Comment:
There is no header so you do not need a title for this screen. Please remove it.